lib/core/Happy.js

```diff
@@ -15,12 +15,15 @@ module.exports = class Happy extends EventEmitter {
   constructor(opts) {
     super();
     this.options = Object.assign({
-      escalationSoftLimitMin: 20, // 20sec
-      escalationSoftLimitMax: 300, // 5min
+      escalationSoftLimitMin: 60, // 1min
+      escalationSoftLimitMax: 600, // 10min
```

Comment:
Is this going to be a major bump since the defaults are changing?

Reply:
yes